lib/extensions/_extensions.scss

```diff
@@ -0,0 +1 @@
+$govuk-extensions-url-context: "/extension-assets"; @import "/Users/matthewmascord/projects/hmrc/tracking-consent-prototype/node_modules/govuk-frontend/govuk/all.scss";
```

Comment:
Fully qualified path used, it should be relative.

Reply:
Looking deeper at it - this should be git ignored as it's a generated file.  For now please add it to `.gitignore` and I'll contact GDS.